src/Core/Services/Implementations/StripePaymentService.cs

```diff
@@ -1128,8 +1050,7 @@ public async Task<string> PayInvoiceAfterSubscriptionChangeAsync(ISubscriber sub
         return paymentIntentClientSecret;
     }
 
-    public async Task CancelSubscriptionAsync(ISubscriber subscriber, bool endOfPeriod = false,
```

Comment:
⚠️ The other side of my comment earlier. This is not "account delete"-related.

Reply:
Agreed, this seems to be a non-issue given that you're removing the single use of this parameter.